<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateAttendee</class>
 <widget class="QDialog" name="CreateAttendee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_password">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_APass">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>259</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AFirst">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>329</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_lastname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Last name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_firstname">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>First name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>0</y>
     <width>851</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600;&quot;&gt;UCCS Registration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AUsername">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>189</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_email">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>471</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Email address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ALast">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>399</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AEmail">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>470</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Submit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
